docs/ppl/datatypes.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Data Types
+parent: Piped Processing Language
+nav_order: 6
+---
+
+
+# Data Types
+
+The following table shows the data types supported by the PPL plugin and how it maps to Elasticsearch and SQL data types:
+
+| PPL Type | Elasticsearch Type | SQL Type
```

Comment:
Table spacing and pipes are a bit odd, but I guess as long as it renders OK. I don't think you need the extra pipe here before PPL Type.

Reply:
Fixed.